crescent/ext/tasks/loop.py

```diff
@@ -0,0 +1,37 @@
+from typing import Callable, Sequence
+
+from crescent.ext.tasks.task import TaskCallbackT, _link_task, _Task
+from crescent.internal import MetaStruct
+
+__all__: Sequence[str] = ("loop",)
+
+
+class _Loop(_Task):
+    def __init__(self, callback: TaskCallbackT, *, hours: int, minutes: int, seconds: int) -> None:
+        self.timedelta = hours * 3600 + minutes * 60 + seconds
+        self.first_loop = True
+
+        super().__init__(callback)
+
+    def time_to_next(self) -> float:
```

Comment:
Just a thought - would `next_iteration` be a better name?

Reply:
That doesn't include `time` so I think its less descriptive.

Reply:
`next_iteration_seconds`?

Reply:
Imo `time_to_next` is much more concise.

Reply:
Maybe `seconds_to_next`? Bikeshedding moment.